packages/types/src/primitive/types.ts

```diff
@@ -48,3 +48,6 @@ export type ExecutionAddress = Bytes20;
 export type RootHex = string;
 /** Non-spec type to signal time is represented in seconds */
 export type TimeSeconds = number;
+export type ColumnIndex = UintNum64; // in reality this number is u8 and can be treated as a JS Number
```

Comment:
why aren't we using `Uint8` here if it's u8?

Reply:
afaik this type is never serialized, rather only used for its associated typescript type.
I don't think there are any consensus types that are treated as numbers that aren't treated as uint64.

Reply:
yeah ok seems like spec uses `uint64`, do we need the comment in that case?

Reply:
Yes, its spec drive as `u64`

Reply:
Agreed. We should get rid of the comments because its spec-specified as `u64`
```suggestion
export type ColumnIndex = UintNum64;
```